index.d.ts

```diff
@@ -3001,11 +3001,11 @@ declare module sequelize {
   }
 
   export interface OrOperator {
-    $or: WhereOperators | WhereAttributeHash | Array<WhereOperators | WhereAttributeHash>;
+    $or: WhereOperators | WhereAttributeHash | Array<Array<number> | WhereOperators | WhereAttributeHash>;
```

Comment:
It's of limited use if it only fixes it for TS 2.4 if the user is using the operator on a number, but not on e.g. strings. I would use the `any` type here then